contrib/python/README.md

```diff
@@ -7,9 +7,33 @@ See [libpod](https://github.com/projectatomic/libpod)
 
 ###
 
-To build the podman wheel:
+To build the podman egg:
 
 ```sh
 cd ~/libpod/contrib/pypodman
-python3 setup.py clean -a && python3 setup.py bdist_wheel
+python3 setup.py clean -a && python3 setup.py bdist
+```
+
+Code snippets/examples:
+
+```python
+"""Show images in storage."""
```

Comment:
I'd rather move these into subheadings.  There's no need to format them like code, and subheadings would get us automatic HTML anchors.  Something like:

```markdown
## List images

``python
import podman

with podman.Client() as client:
    list(map(print, client.images.list()))
``

## List containers created since midnight

``python
from datetime import datetime, time
...
``
```

Except with triple backticks instead of double for the Python blocks.